src/repl-frontend/scala/tools/nsc/interpreter/shell/InteractiveReader.scala

```diff
@@ -34,6 +32,8 @@ trait InteractiveReader {
     // hack necessary for OSX jvm suspension because read calls are not restarted after SIGTSTP
     if (scala.util.Properties.isMac) restartSysCalls(readOneLine(prompt), reset())
     else readOneLine(prompt)
+
+  def initCompletion(completion: Completion): Unit = {}
```

Comment:
I don't think sbt used `postInit`, if that's what you're concerned about? I in-sourced its console interface to be sure.

Are you saying the unit saturation levels are sufficient, or are you encouraging more of it? If so, how? :-)